core/unit_test/hip/TestHIP_Spaces.cpp

```diff
@@ -176,11 +275,19 @@ TEST(hip, space_access) {
           Kokkos::HostSpace>::accessible,
       "");
 
+#if defined KOKKOS_ENABLE_HIP_UNIFIED_MEMORY
```

Comment:
I think this `ifdef` condition should be moved so that `static_assert` for `HIPHostPinnedSpace` accessibility can still run.

Reply:
you are perfectly right, resolved in [10a62dc](https://github.com/kokkos/kokkos/pull/5112/commits/10a62dcca598f086cd49a542f579610b0e6eb118)